utils/project.go

```diff
@@ -47,7 +47,13 @@ func DetermineProjectInfo(projectPath string) (string, string) {
 		language = "swift"
 		buildType = "swift"
 	}
-	return language, buildType
+	if _, err := os.Stat(path.Join(projectPath, "stack.yaml")); err == nil {
```

Comment:
The `PathExists` function was recently added to `filesystem.go` and other functions in `project.go` use it. Can you switch to that?

Reply:
Done